src/Umbraco.Web/Security/UmbracoBackOfficeIdentity.cs

```diff
@@ -204,17 +203,17 @@ private UmbracoBackOfficeIdentity(ClaimsIdentity identity)
         private string[] _allowedApplications;
         public string[] AllowedApplications => _allowedApplications ?? (_allowedApplications = FindAll(x => x.Type == Constants.Security.AllowedApplicationsClaimType).Select(app => app.Value).ToArray());
 
-        public int Id => int.Parse(this.FindFirstValue(ClaimTypes.NameIdentifier));
+        public int Id => int.Parse(this.FindFirst(ClaimTypes.NameIdentifier)?.Value);
```

Comment:
We could consider making an extention method for `FindFirstValue`, to ensure we dont forget the null check etc.